demisto_sdk/commands/validate/validators/GR_validators/GR101_is_using_invalid_from_version.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
+from demisto_sdk.commands.content_graph.objects import (
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+    Classifier,
+    CorrelationRule,
+    Dashboard,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    Integration,
+    Job,
+    Layout,
+    LayoutRule,
+    Mapper,
+    ModelingRule,
+    Pack,
+    ParsingRule,
+    Playbook,
+    Report,
+    Script,
+    TestPlaybook,
+    Trigger,
+    Widget,
+    Wizard,
+    XSIAMDashboard,
+    XSIAMReport,
+)
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    Integration,
+    Script,
+    Playbook,
+    Pack,
+    Dashboard,
+    Classifier,
+    Job,
+    Layout,
+    Mapper,
+    Wizard,
+    CorrelationRule,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    LayoutRule,
+    Layout,
+    ModelingRule,
+    ParsingRule,
+    Report,
+    TestPlaybook,
+    Trigger,
+    Widget,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    XSIAMDashboard,
+    XSIAMReport,
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+]
+
+
+class IsUsingInvalidFromVersionValidator(BaseValidator[ContentTypes], ABC):
+    error_code = "GR101"
+    description = "Validates that source's fromversion >= target's fromversion."
+    rationale = (
+        "Prevent issues where used objects are not available due to a version mismatch."
+    )
+    error_message = (
+        "Content item '{0}' whose from_version is '{1}' is using content items:"
+        " {2} whose from_version is higher (should be <= {3})"
```

Comment:
Is there a reason for the double quotes?

```suggestion
            error_message = (
        "Content item '{0}' whose from_version is '{1}' is using content items:
        {2} whose from_version is higher (should be <= {3})"
    )
```

Reply:
Python's syntax, since this isn't a [multiline string](https://www.w3schools.com/python/gloss_python_multi_line_strings.asp).